vuln-reach/src/javascript/lang/accesses.rs

```diff
@@ -165,17 +165,23 @@ impl<'a> AccessGraph<'a> {
     /// NOTE: `test` in this example is not declared as variable and thus
     /// hoisted to the global scope. This is why the accessor of `bar` cannot be
     /// `foo`, otherwise the access from `baz` to `bar` would not be detected.
-    fn find_accessor(cursor_cache: &mut TreeCursorCache<'a>, node: Node<'a>) -> Option<Node<'a>> {
+    fn find_accessor(
+        cursor_cache: &mut TreeCursorCache<'a>,
+        mut node: Node<'a>,
+    ) -> Option<Node<'a>> {
         let cursor = cursor_cache.cursor(node).unwrap();
+        let mut cur_depth = 0;
```

Comment:
I think this is a bad name. Current depth is `depth` itself. This is more akin to the origin/target of the function.